packages/components/src/components/brand/brand.scss

```
@@ -13,7 +13,7 @@
 	&::before {
 		--db-icon-font-size: #{variables.$db-sizing-md};
 		--db-icon-margin-end: 0;
-		--db-current-icon-color: #{colors.$db-brand-origin-enabled};
+		color: #{colors.$db-brand-origin-enabled};
```

```suggestion
		--db-icon-color: #{colors.$db-brand-origin-enabled};
```

why not use the existing CSS Custom property for this one ?